.env_example

```
@@ -0,0 +1,26 @@
+# #########
+# General deployment parameters
+###########
+
+DEPLOYER_PRIV_KEY = <DEPLOYER_PRIVATE_KEY>
+VOLTA_CLAIM_MANAGER_ADDRESS = "0x5339adE9332A604A1c957B9bC1C6eee0Bcf7a031"
+STAKINGPOOL = "0x181A8b2a5AEb25941F6A79b4aE43dBb1968c417A"
+INITIATOR_ADDRESS = <ADMIN_ADDRESS>
+CLAIM_MANAGER_ADDRESS = "0xC3dD7ED75779b33F5Cfb709E0aB02b71fbFA3210"
```

Let's correct this value as well to be the correct EWC value
```suggestion
CLAIM_MANAGER_ADDRESS = "0x23b026631A6f265d17CFee8aa6ced1B244f3920C"
```

---

implemented in commit [3be9e0a](https://github.com/energywebfoundation/staking-pool/pull/61/commits/3be9e0aa2c923b0d0a2caeeee502616eb585843f)